### PR TITLE
Added to About Page

### PR DESCRIPTION
**Pull request by Dravani**
Issue 15: Implement About Page

Added Descriptions, Images, and Personal Info of the Team. If you would like me to change anything, let me know.

### CODE REVIEW

File: my-ai-stock-buddy/src/pages/AboutPage.tsx [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
 const AboutPage: React.FC = () => {
 
     return (
       <>
         <Nav />
-        <p>This is the About page.</p>
+        <div className='About-Page'>
+          <img src = '/images/stockMarketImage.jpg' alt='About Us' className='About-Image'/>
+          <h1 className = "About-Header">About Stock Buddy</h1>
+          <p className='About-Summary'>Stock Buddy is a comprehensive stock market anaylsis tool developed by a group of dedicated college students. Our goal is to
```
Comment (rileycleavenger) [inline my-ai-stock-buddy/src/pages/AboutPage.tsx:15]: a few typos in here

File: my-ai-stock-buddy/src/components/TeamMemberCard.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+const TeamMemberCard: React.FC<TeamMemberCardProps> = ({ member }) => {
+  return (
+    <div className="team-member">
+      <img src={member.image} alt={member.name} className="team-member-image" />
+      <h3>{member.name}</h3>
+      <p>{member.role}</p>
+      <div className="team-member-links">
+        <a href={member.linkedin} target="_blank" rel="noopener noreferrer">LinkedIn</a>
```
Comment (rileycleavenger) [inline my-ai-stock-buddy/src/components/TeamMemberCard.tsx:15]: I think these would be better as logo icons

**Review by rileycleavenger — CHANGES_REQUESTED** [review]
image

For smaller screens you should have the ones that drop down to the next line center in the second row. Also fix the typos, add a slight margin or padding to the top of the page, and switch the "LinkedIn" and "GitHub" to icons (for that you can probably use font awesome or something similar). Other than that it looks really good!

**Review by dos-2 — APPROVED** [review]
Once you've completed riley's changes and get his approval, you're good to go!

**Review by rileycleavenger — APPROVED** [review]
LGTM!

**Pull request merged**